guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,40 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Verify Content Checksum on container repositories using Hammer
+Option '--validate-contents' now supports all repository types, using "hammer repository synchronize" command. 
+Previously container repositories and some others were not supported.
+
+=== Add ACS bulk refresh option
+Added option '--all' to "hammer alternate-content-source refresh" command.
+
+=== Experimental activation key details page
+When the setting 'Show Experimental Labs' is enabled, the Activation Key details page will be replaced by the beginnings of a redesigned page.
+
+=== Content Views and CVV enhancements to filters and refresh
+Content View filter creation/edit UI: 'Edit Rule' button is now 'Save'.
+If dependency solving is enabled or disabled for a content view, it will now be marked as needing a new publish.
+
+=== Pulpcore and plugins updated from Pulpcore 3.22 to 3.28
+Contains both fixes and features that Katello can use in the future to better manage content, such as:
```

Comment:
I think you need an empty line in between:
```suggestion
Contains both fixes and features that Katello can use in the future to better manage content, such as:

```

Reply:
In latest push